modules/host/storageobligations.go

```diff
@@ -372,6 +372,7 @@ func (h *Host) addStorageObligation(so *storageObligation) error {
 
 	// Update the host financial metrics with regards to this storage
 	// obligation.
+	h.financialMetrics.ContractCount++
```

Comment:
seems a little odd to maintain the current ContractCount in memory, but load it from `Stats().KeyN`...
I dunno, it's fine.